core/commands/config.go

```diff
@@ -41,15 +41,15 @@ repository.`,
 much like 'git config'. The configuration values are stored in a config
 file inside your IPFS repository.
 
-EXAMPLES:
+Examples:
 
 Get the value of the 'datastore.path' key:
 
-  ipfs config datastore.path
+  $ ipfs config datastore.path
```

Comment:
Are you sure this is a good idea? I have seen people copy and paste these things including the `$` and then wonder why it's not working.